Ignore blank lines and lines starting with '#'

kellybyrd opened the pull request:
Sorry for the two unrelated fixes in one PR, I didn't create a PR branch for these:

1) In CSV input file, ignore blank lines and lines starting with '#'. When adding a lot of user-defined sounds, it is convenient to break up the file into sections using blank lines and add comments. It also makes it easy to comment out the "hello.wav" sound, to avoid a loud volume OpenTX startup.

2) Be more specific with output format in 'say' command. I had a problem with the Allison voice producing wave files that Companion and VLC would play, but my TX would not. Using the "file" command and comparing against existing wav files on my SD card for other languages, there were some differences, so I added the --file-format, cmdline param. It shouldn't be needed, but it is for me.

Commit:
Ignore blank lines and lines starting with '#'

When adding a lot of user-defined sounds, it is convenient to break
up the file into sections using blank lines and add comments.

RCdiy commented on Mac Apps Scripts/README.md:
```diff
@@ -32,12 +32,13 @@ info@RCdiy.ca
 ### Examples
  SOUNDS/en/SYSTEM;0000.wav;0
 
+ # This is a comment line
```
This file format is defined by the OpenTX developers which is why I want to stick to what we have.

RCdiy replied:
But on the other hand I like your changes. Let me think about it:)

RCdiy commented on Mac Apps Scripts/SoundPackGenerator.sh:
```diff
@@ -80,7 +80,7 @@ else
   fi
 
-  awk -F ";" -v Voice=$Voice '{
+  awk -F ";" -v Voice=$Voice 'NF && $1!~/^#/ {
```
Could you keep both the old and new on seperate lines. Comment out the old.

Commit:
Be more specific with output format in 'say' command

With some voices on OSX, I have to be specific with the output format
when using the say command or I end up with wav files that won't
play on my TX. This was happenning with:
- OSX 10.13.2, the "Allison" voice, I think I upgraded it to high-quality
- OpenTX 2.2.1
- A production Horus TX

RCdiy commented:
I’ll look at this when I can get to a decent internet connection. On vacation. Thanks for the modifications. Could you comment out the old line and make a new line with your modifications?

RCdiy committed:
Update SoundPackGenerator.sh

RCdiy committed:
Update SoundPackGenerator.sh

RCdiy commented:
Thanks:)

kellybyrd commented:
Perfect, thanks!
